server/remote_cache/config/config.go

```diff
@@ -2,8 +2,20 @@ package config
 
 import "flag"
 
-var zstdTranscodingEnabled = flag.Bool("cache.zstd_transcoding_enabled", true, "Whether to accept requests to read/write zstd-compressed blobs, compressing/decompressing outgoing/incoming blobs on the fly.")
+var (
+	zstdTranscodingEnabled           = flag.Bool("cache.zstd_transcoding_enabled", true, "Whether to accept requests to read/write zstd-compressed blobs, compressing/decompressing outgoing/incoming blobs on the fly.")
+	unconditionallyCompressWrites    = flag.Bool("cache.unconditionally_compress_writes", false, "EXPERIMENTAL: Whether to compress uncompressed write requests before forwarding to remote cache.")
```

Comment:
Looks like this flag is only used in byte_stream_server_proxy, so I'd recommend just moving it there for now. It's easy enough to move to a shared location later if necesssary.

Reply:
Also, I'd recommend calling this flag and the below `cache_proxy.flag_name` because there will likely only be used in the proxy (I think).